index.js

```diff
@@ -644,28 +641,6 @@ Client.prototype.transition = function (newPaymentId, callback) {
     if (err) return callback(err)
 
     if (!body.unsignedTx) return callback(new Error('expecting an unsignedTx'))
```

Comment:
@jasonrsadler you can remove this entire `transition` function - it is no longer used in `browser-laptop`

Reply:
Done. There is also 'transitioned ()'. Does that go as well or stay?